packages/venia-ui/lib/components/LegacyMiniCart/productOptions.js

```diff
@@ -15,11 +15,15 @@ const ProductOptions = props => {
                 const key = `${option_label}${value_label}`;
 
                 return (
-                    <Fragment key={key}>
-                        <dt className={classes.optionLabel}>
-                            {option_label} : {value_label}
-                        </dt>
-                    </Fragment>
+                    <div key={key} className={classes.optionLabel}>
```

Comment:
Refactored this piece of code because the correct way to represent a `<dl />` tag is to have a `<dt />` and a `<dd />` tag. Our old implementation only has the `<dt />` tag.

Had to wrap both of them in a `div` instead of a `Fragment` because the fragment component does not honor the CSS given to it. Used `div` because it is a valid child of the `dl` tag.